dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -1078,7 +1076,7 @@ function onGameStateChange(msg) {
   updateDonationStatus(state.donation_active);
 
   // Update gridItems
-  if (state.items !== undefined && state.items !== null) {
+  if (! _.isNil(state.items)) {
```

Comment:
I think this is `isNull()` ? https://underscorejs.org/#isNull

Reply:
We're using lodash, so it's [isNil](https://lodash.com/docs/4.17.15)